package.json

```diff
@@ -4,7 +4,7 @@
   "description": "amplify-codegen",
   "scripts": {
     "test-changed": "lerna run test --since main",
-    "test": "lerna run test",
+    "test": "lerna run test -- -u",
```

Comment:
This change will override the snapshots in CI without commits and should be removed

Reply:
Fixed